infrastructure/requirements.txt

```diff
@@ -1,5 +1,8 @@
-aiogram~=3.0
```

Comment:
why remove aiogram? sometimes you'll need to have a bot for TG notifications

Reply:
ahh, I see, I am going to fix it